tests, multus, sriov: Refactor tests

oshoval opened the pull request:
Split the helper function to two parts,
according their logic:
1. Creating the VMs
2. Pinging (which is done from the test now)

Signed-off-by: Or Shoval <oshoval@redhat.com>

```release-note
None
```

oshoval commented:
/assign @EdDev 
Its the refactor we removed from #4335

EdDev reviewed (approved):
Thanks.

Note that you better add the "why" in the commit next time.

I do not know why you have not left this in the original PR as a pre-commit.
But if you prefer to wait more time and do rebases, I do not mind.
(it does have the advantage to be reviewed/approved faster)

oshoval commented:
> Thanks.
> 
> Note that you better add the "why" in the commit next time.
> 
> I do not know why you have not left this in the original PR as a pre-commit.
> But if you prefer to wait more time and do rebases, I do not mind.
> (it does have the advantage to be reviewed/approved faster)

Thanks
The only why now is to split it logically and this what i wrote,
i removed it from the other PR as it doesnt belong there, not needed at all there,
it was just if we wanted to keep the set via console because then we would need the status to be stable, and on 
the way i added the name checking, but since the PR of network remove the need of this function there, we dont want
to put this logic of network name there.
We cant even move the skip anymore to a later place in the function, because it will flake until the network data PR is merged.

oshoval commented:
/cc @AlonaKaplan 
Please review when you can

AlonaKaplan commented:
/approve
